ouroboros-network/testlib/Test/Ouroboros/Network/Diffusion/Node.hs

```diff
@@ -114,6 +114,8 @@ import Ouroboros.Network.Snocket (MakeBearer, Snocket, TestAddress (..),
 
 import Simulation.Network.Snocket (AddressType (..), FD)
 
+import Ouroboros.Network.Diffusion.Types (DiffusionArguments (..),
+           DiffusionConfiguration (..))
```

Comment:
Everything that's needed should be exported from `Ouroboros.Network.Diffusion` module.
```suggestion
import Ouroboros.Network.Diffusion qualified as Diff
```